mission/functions/core/init/fn_curator_init_client.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+    File: fn_curator_init_client.sqf
```

Comment:
`init_client` isn't really accurate... as this never runs on the client, it's all running server side?

also, if it's all running server side, why do we even need to `remoteExec` the assign script on the server? we're already executing on the server?!